Develop

KolodnytskaV opened the pull request:
[DEMO LINK](https://KolodnytskaV.github.io/layout_landing-page/)

KolodnytskaV committed:
changes

KolodnytskaV committed:
page

Anton-Kuchmasov reviewed (changes requested):
Good job!
However, there are a few issues need to be fixed:

1. When I'm sending a Form, there should be only `reset` action. Check #15 in this [instruction](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md)
<img width="1918" height="869" alt="image" src="https://github.com/user-attachments/assets/8de0c524-8f00-43b3-8621-4a11280bda35" />

2. All contact data in footer (phone number, address, email) should be clickable with some actions. Also please add some effects on `hover` for them

3. This block (screenshot) looks different, not the same with the Figma mockup:
<img width="1919" height="505" alt="image" src="https://github.com/user-attachments/assets/4d4739ee-1ca0-4c60-a5a2-6a24f17ad4a1" />

4. Would you mind adding `cursor: pointer` to all clickable elements (e.g. links, buttons)?
<img width="1919" height="919" alt="image" src="https://github.com/user-attachments/assets/8e08e7ee-3728-4f65-8f52-b0a14d40c424" />

5. Add some interactive to these cards:
<img width="1861" height="924" alt="image" src="https://github.com/user-attachments/assets/7a118f1c-a605-4c15-8efa-17343e27f82b" />

6. Styles in header also looks different (double-check paddings)
<img width="1918" height="326" alt="image" src="https://github.com/user-attachments/assets/6b88fd86-ffb5-4995-80c7-2837551823d6" />


7. 
6. 
5. 
4.

KolodnytskaV committed:
changes2

KolodnytskaV commented:
Hello, I have fixed some issues that you send me (1, 2, 4, 5), but  I can't fixe a size of logo in top__bar. Timeless section is good on my desktop (the same with a Figma mockup). Section contact for devices >1240px has a different padding but I can't understand how to fix it. I also added cursor: pointer for button "View all" but it doesn't work.

etojeDenys reviewed (changes requested):
1. increase the scale of the images on hover

2. add a cursor pointer to the button. if you have any issues, feel free to ask for help in the chat
<img width="414" height="218" alt="image" src="https://github.com/user-attachments/assets/d3a92d7a-3970-49e2-baff-b776c8f19ccd" />

3. remove the line under the links and add a hover effect
<img width="268" height="328" alt="image" src="https://github.com/user-attachments/assets/38f13d4c-e361-4e87-95f7-984008dec800" />

KolodnytskaV committed:
changes3

KolodnytskaV committed:
changes for new review

vitalii-bondarenko-ua reviewed (changes requested):
Good job. Almost done.
- check all hover effects. Hover should be smooth on both sides. For fix it set transition to selector withour :hover

<img width="1903" height="761" alt="Screenshot 2025-10-23 at 15 40 45" src="https://github.com/user-attachments/assets/48a246b6-a602-4312-aea1-a9675fc6924e" />

- content in contact us form not centered.
<img width="1903" height="723" alt="Screenshot 2025-10-23 at 15 39 06" src="https://github.com/user-attachments/assets/4872c0eb-dacb-4095-a67f-3d9ff0dadd0a" />

- Change text color on hover for phone, email, and address

KolodnytskaV committed:
change color text contacts

KolodnytskaV committed:
changes color

KolodnytskaV committed:
center contact section

Anton-Kuchmasov reviewed (changes requested):
Hey, your solution still isn't perfect. Check out chat in fe_chat, I gave you a couple of suggestions

KolodnytskaV committed:
remove img hover

KolodnytskaV committed:
top-bar changes

etojeDenys reviewed (changes requested):
1. decrease the scale size a little because now, when the image is hovered, it looks like it merges with the other images
<img width="426" height="269" alt="image" src="https://github.com/user-attachments/assets/5565733a-2412-4339-96ae-dc9c0744b652" />

2. your buttons should have the same style and follow this layout
layout: 
<img width="575" height="125" alt="image" src="https://github.com/user-attachments/assets/57e60cfd-c838-42aa-b567-1aafd82dc2a7" />

3. on the desktop version, the phone number should appear when the user hovers over the icon. also it should be clickable tel link
<img width="225" height="74" alt="image" src="https://github.com/user-attachments/assets/f80d98d4-c884-48d2-ae98-fd3ce15c1c43" />

4. page looks broken at this scale
<img width="656" height="749" alt="image" src="https://github.com/user-attachments/assets/9ffaa5ad-1f6c-4b1e-a014-53e90cbe5b38" />

KolodnytskaV committed:
button changes + phone_icon

KolodnytskaV commented:
I've changed a style of button "view all", I also added a phone number for phone__icon. I tried to change a scale for images in section sound but it doesn't work. I also don't know how to fix an image for section timeless

maxim2310 reviewed (changes requested):
Still have same issues as before, maybe you forgot to deploy. anyway if you have questions you can ask in FE chat

KolodnytskaV committed:
change scale

etojeDenys reviewed (changes requested):
1. phone number should show up when you hover icon and should have a different style
<img width="345" height="65" alt="image" src="https://github.com/user-attachments/assets/836a6f08-f452-4d17-9922-3c732997c25e" />

2. send button should have the same style as "View all," except for its width

KolodnytskaV committed:
changes button send like button view all + phone number ok

etojeDenys reviewed (approved):
good job
